docs/book/installation.md

```diff
@@ -1,5 +1,63 @@
-**This is only a placeholder.**
+---
+show_file_content: true
+---
 
-The content of this page can be found under:
+## Avoid unused cache adapters are being installed
+With `laminas-cache` v2.10.0, we introduced satellite packages for all cache adapters.
 
-https://github.com/laminas/documentation-theme/blob/master/theme/pages/installation.html
\ No newline at end of file
+In case, there is no need for several adapters in your project, you can use composer to ensure these adapters are not being installed. To make this happen, you have to specify a `replace` property within the `composer.json` of your project.
+
+### Example composer.json with only memory adapter being installed
+```json
+{
+    "name": "vendor/project",
+    "description": "",
+    "type": "project",
+    "require": {
+        "laminas/laminas-cache": "^2.10",
+        "laminas/laminas-cache-adapter-storage-memory": "^1.0"
+    },
+    "replace": {
+        "laminas/laminas-cache-storage-adapter-apc": "*",
+        "laminas/laminas-cache-storage-adapter-apcu": "*",
+        "laminas/laminas-cache-storage-adapter-blackhole": "*",
+        "laminas/laminas-cache-storage-adapter-dba": "*",
+        "laminas/laminas-cache-storage-adapter-ext-mongodb": "*",
+        "laminas/laminas-cache-storage-adapter-filesystem": "*",
+        "laminas/laminas-cache-storage-adapter-memcache": "*",
+        "laminas/laminas-cache-storage-adapter-memcached": "*",
+        "laminas/laminas-cache-storage-adapter-mongodb": "*",
+        "laminas/laminas-cache-storage-adapter-redis": "*",
+        "laminas/laminas-cache-storage-adapter-session": "*",
+        "laminas/laminas-cache-storage-adapter-wincache": "*",
+        "laminas/laminas-cache-storage-adapter-xcache": "*",
+        "laminas/laminas-cache-storage-adapter-zend-server": "*"
+    }
+}
+```
+
+> ### Only necessary in 2.10+
+> Starting with 3.0.0, no storage adapter is required by this component and thus, each project has to specify the storage adapters which are required by the project.
```

Comment:
```suggestion
> ### Only necessary in 2.10+
>
> Starting with 3.0.0, no storage adapter is required by this component and thus, each project has to specify the storage adapters which are required by the project.
```

Reply:
Move this block quote to the beginning as it breaks the example and all version 3 users can skip this part / information from the beginning.
(after the headline "Avoid Unused Cache Adapters Are Being Installed")